### PR TITLE
Update ECS to 8.4.0 Part 4

### PR DESCRIPTION
**Pull request by leehinman**
- cylance
- f5
- fim
- fireeye
- fortinet
- gcp
- gcp_pubsub
- github
- google_workspace
- hashicorp_vault

### CODE REVIEW

File: packages/fim/data_stream/event/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ description: Pipeline for processing auditd events
 processors:
   - set:
       field: ecs.version
-      value: 8.3.0
+      value: '8.4.0'
```
Comment (kcreddy) [inline packages/fim/data_stream/event/elasticsearch/ingest_pipeline/default.yml:6]: Missing pipeline test expected log file

File: packages/gcp/data_stream/vpcflow/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ description: Pipeline for Google Cloud VPC Flow Logs
 processors:
   - set:
       field: ecs.version
-      value: '8.3.0'
+      value: '8.4.0'
```
Comment (kcreddy) [inline packages/gcp/data_stream/vpcflow/elasticsearch/ingest_pipeline/default.yml:7]: There seems to be 3 more data_streams under `gcp` namely `billing`, `compute`, and `firestore`. Not sure if they also need to be udpated?

File: packages/gcp_pubsub/manifest.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name: gcp_pubsub
 title: Custom Google Pub/Sub Logs
-version: "1.1.0"
+version: "1.2.0"
```
Comment (kcreddy) [inline packages/gcp_pubsub/manifest.yml:3]: Probably `generic` datastream pipeline inside `gcp_pub` needs ECS version update

File: packages/hashicorp_vault/data_stream/metrics/elasticsearch/ingest_pipeline/default.yml [hunk 4]
```diff
@@ -11,7 +11,7 @@ processors:
       ignore_missing: true
   - set:
       field: ecs.version
-      value: '8.3.0'
+      value: '8.4.0'
```
Comment (kcreddy) [inline packages/hashicorp_vault/data_stream/metrics/elasticsearch/ingest_pipeline/default.yml:14]: Missing pipeline test expected log file for `metrics` data_stream

**Review by kcreddy — CHANGES_REQUESTED** [review]
Just some minor changes (mainly pipeline test files) which could be added. Rest LGTM

**Comment by taylor-swanson** [comment]
I don't think @elastic/obs-cloud-monitoring should be listed as a code owner here, all of these integrations are owned by @elastic/security-external-integrations?

File: packages/fim/data_stream/event/elasticsearch/ingest_pipeline/default.yml (hunk 1 above)
Comment (taylor-swanson) [reply packages/fim/data_stream/event/elasticsearch/ingest_pipeline/default.yml:6]: Adding tests is out of scope for this type of change and should be handled separately. This change only involves updating the ECS version.

File: packages/gcp/data_stream/vpcflow/elasticsearch/ingest_pipeline/default.yml (hunk 2 above)
Comment (taylor-swanson) [reply packages/gcp/data_stream/vpcflow/elasticsearch/ingest_pipeline/default.yml:7]: Those look to be metrics data streams, which don't have ingest pipelines (nor expected files to update).

File: packages/hashicorp_vault/data_stream/metrics/elasticsearch/ingest_pipeline/default.yml (hunk 4 above)
Comment (taylor-swanson) [reply packages/hashicorp_vault/data_stream/metrics/elasticsearch/ingest_pipeline/default.yml:14]: Same comment above regarding metrics data streams.

**Review by taylor-swanson — APPROVED** [review]
LGTM

File: packages/fim/data_stream/event/elasticsearch/ingest_pipeline/default.yml (hunk 1 above)
Comment (kcreddy) [reply packages/fim/data_stream/event/elasticsearch/ingest_pipeline/default.yml:6]: Got it, thanks for letting me know 👍🏼

File: packages/gcp/data_stream/vpcflow/elasticsearch/ingest_pipeline/default.yml (hunk 2 above)
Comment (kcreddy) [reply packages/gcp/data_stream/vpcflow/elasticsearch/ingest_pipeline/default.yml:7]: Makes sense

**Review by kcreddy — APPROVED** [review]

**Pull request merged**